src/main/java/mate/academy/spring/controller/OrderController.java

```
@@ -34,14 +34,16 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        ShoppingCart cart = shoppingCartService.getByUser(userService
+                .findByEmail(authentication.getName()).get());
```

let's throw an exception in case user isn't found, the same for other similar cases